### ISSUE TITLE
Latest commit deleted ~8000 lines of translations

### ISSUE THREAD

**Issue by StefanErakovic** [Magento2Translations/language_el_gr#1]
Seeing as how magento composer update pulled this commit, it's a big issue.

**Comment by MrGekko** [comment]
This is very strange I have no idea why this happens... Could you please install one of the version branches... they seem to be correct.

**Comment by StefanErakovic** [comment]
Yeah I did that on my install, just thought I let you know of this strange bug.
